src/interfaces.js

```diff
@@ -132,5 +132,5 @@ interface ConfigureRouter {
   /**
   * Implement this hook if you want to add sub-routes to your view.
   */
-  configureRouter(config: RouterConfiguration, router: Router);
+  configureRouter(config: RouterConfiguration, router: Router): any;
```

Comment:
I don't see what's the benefit of doing union with `any`? The compiler will never throw an error. It's as good as a doc comment. Or am I missing something?